Library/Formula/hue.rb

```diff
@@ -0,0 +1,94 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  depends_on :java
+  depends_on :mysql
+
+  depends_on "gmp"
+  depends_on "maven" => :build
+  depends_on "openssl"
+
+  # Patches have been merged upstream.  Should try building next release
+  # (> 3.8.1) without them.
+  #
+  # Syntax error in 'parquet-python' library:
+  #   https://github.com/cloudera/hue/pull/206
```

Comment:
Would it be possible to use `https://github.com/cloudera/hue/pull/206.patch` and `https://github.com/cloudera/hue/pull/207.patch` as `patch do` URLs instead?